.vsts-ci.yml

```diff
@@ -0,0 +1,89 @@
+trigger:
+- master
```

Comment:
what does this mean? Can you add a comment?

Reply:
`trigger` specifies which branch[es] would trigger the CI run, as per https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=vsts#trigger
we also run CI on PRs to `master`.

do you believe that this should go into the `md` file?

Reply:
All branches should trigger a CI run.

Reply:
Cool. Will adjust. Below is a link to the short document that I made about the setup:

https://docs.google.com/document/d/1GlxrpRfw7VClScYFQk5H_y5fUhA6kIuidbbx2I0cBio/edit?usp=sharing